gestalt-inject-java/src/main/java/org/terasology/gestalt/annotation/processing/ElementUtility.java

```diff
@@ -11,23 +11,41 @@
 import javax.lang.model.util.Types;
 import java.util.List;
 
+/**
+ * BenjaminAmos, pls proper description.
```

Comment:
```suggestion
 * Utilities for discovering and distinguishing element types.
```

This is a utility class: convenient but just a mixture of related useful routines. Most projects have one somewhere, even if it's not considered the best programming practice sometimes.